### ISSUE TITLE
[Policies] Input data form 

### ISSUE THREAD

**Issue by yatil** [w3c/wai-policies-prototype#306]
_From @sharronrush on January 13, 2017 19:7_

Please comment on the proposed data input fields and approach to gather data on existing policies and updates. The [Policy_update_form.docx](https://github.com/w3c/wai-policieslist/files/705120/Policy_update_form.docx) proposal was sent to the EOWG list and is attached here.  

_Copied from original issue: w3c/wai-policieslist#15_

**Comment by yatil** [comment]
_From @nitedog on January 16, 2017 16:49_

On this page I think it is good to have separate "WCAG version used" and "Standard used" entries, because the WCAG entry is a single-option (to keep the data consistent). The form looks good to me.

**Comment by yatil** [comment]
_From @lakeen on January 17, 2017 18:50_

The form input fields seem complete, except for possibly a date field after the status field.

**Comment by yatil** [comment]
_From @slhenry on January 18, 2017 1:35_

Great start! Thanks!
Some thoughts:
1. Overall
1.1. Required fields – I think we want to make very few things required fields – maybe only Country. Sure ideally the submitter would enter all the info; however, they might not know all of it – and not be willing to figure it out. I think we would rather have incomplete data that no data at all. (e.g., there are too many required fields so they just don't submit anything)
 (We can figure out how much we publish incomplete data versus do the research ourselves or get others to fill it in.)
1.2. Order – Consider the order of the fields. I suggest putting the most important first – that way if the submitted runs out of time or inclination to do all the fields, we at least get the most important info.
1.3. Controls – Checkbox or option/radio button lists are much easier to use than drop-downs. I think we want to use lists in most cases.
1.4. Descriptions – We should include descriptions with the fields. (They're drafted in the other doc… although it looks like the descriptions don't match the options, but we can clean that up later. :-) We want to make it easy for anyone to submit info, not just people familiar with it.
2. Type
2.1. Categories - I think these categories need rethinking and/or we need to make it multi-select. For example, if something is a Proposed Procurement Law, it is three of these options.
2.2. Other|dunno - I said in the meeting that we probably needed "I don't know" and/or "Other" -- although if this field is not required, then maybe we don't needs this – or, maybe we want to give it to them anyway so they are encouraged to fill out more fields? Also, do we need or want to have Other with a text field? Possibly something(s) typed in there might become new categories that we hadn't thought of.
2.3. Recommendation – Given that "Recommendation" has a specific meaning in W3C-land, and it's not particularly clear to many people, it's best if we can avoid using that word in other contexts.
3. Status – For "Approved, not in effect" how about adding the date(s) that it does go into effect?
4. Scope – Can we have 3 multi-select checkboxes for this?
5. Web only – Why do we want this info? Do we want to know and show other related info? e.g., Web, mobile, software, hardware? Web content, web authoring tools, web user agents?
6. WCAG version
6.1. Do we want to be more specific like "references WCAG 2.0 directly", or "exact translation of WCAG 2.o"?
6.2. Maybe add to "WCAG 2.0 derivative": "- mostly based on WCAG 2.0 with some changes?
6.3. Maybe "Not used" -> "Does not use WCAG"
6.4. TBD ? Maybe: "Draft policy, standard not yet determined"?
7. Standard
7.1. I actually stumbled on this and didn't know what it was asking at first. Maybe "Standard used" -> "Other Standard used". Or maybe have a fieldset labeled "Standard Used" that includes a question for WCAG version" and another or "Other Standard"? Or maybe just one question "Standard Used" with all those WCAG options, and then in the same list an "Other" option with a text field. (That seems simplest, but maybe I'm missing something? :-)
7.2. I wish there were policies that use ATAG & UAAG! :-/
8. Relevant documents – Would be good to give some guidance about the types of docs we want here and do not want here…

**Comment by yatil** [comment]
_From @bakkenb on January 18, 2017 21:56_

Wow Shawn, you covered it really well. I had a few of the same questions, but I will not restate them for the sake of Mary Jo's sanity in reading through the feedback. 

With Shawn's comments, I agree with most, but I particularly agree with:
1.1 - agree that not everyone will have all the detail up front
1.4 - I was wishing I had descriptions for the categories so that I could understand them better. That will be the same for users
2.3 - agree
3 - agree it would be helpful to have a date if they know it
6.3 and 6.4 - add more detail for clarity
8 - smart to give some guidance. Especially about what is *not* wanted

Other comments from me...
- Didn't really understand the full idea of "Web Only." I think that needs a bit more clarification or description.
- If they add a country, and then add a province/state, will there be a different set of input fields for "state type" input? Or will the content be the same? If they fill out the rest of the input fields, how will we know whether the information provided is for the country in general or for the specific state?

**Comment by iamjolly** [comment]
I don't know if this is still relevant for the current submission form hosted at:

https://www.w3.org/WAI/Policy/submission.html (existing design) and
https://w3c.github.io/wai-policies-prototype/policies/submission/ (new design)

But, there is a need for some styling updates for the form in the new design which will be opened with separate issue(s). 

I recommend closing this issue for now and opening a new issue or issues with specific changes to be made to the form fields, if any, that must happen before the re-launch.
